server/src/index.ts

```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env node
 /* istanbul ignore file: setup file */
-import "module-alias/register";
```

Comment:
How are the module aliases working with this gone?

My mental model was we needed to register a loader, ideally at the earliest point, that could override with the aliases?

Reply:
Tested and the removal of this doesn't seem to stop it functioning. Maybe something has changed in `module-alias`.